connector/src/main/java/org/geysermc/connector/GeyserConnector.java

```diff
@@ -524,6 +518,15 @@ public boolean isProductionEnvironment() {
         return !"DEV".equals(GeyserConnector.VERSION);
     }
 
+    /**
+     * Deprecated. Get the AuthType from the GeyserConfiguration through {@link GeyserConnector#getConfig()}
+     * @return The
+     */
+    @Deprecated
+    public AuthType getDefaultAuthType() {
+        return getConfig().getRemote().getAuthType();
+    }
```

Comment:
could also make a deprecated setter but 🤷

Reply:
I'm not even sure if we need this as long as we check for our projects' usage.

Reply:
I'll check GeyserConnect and Geyser-Fabric but I don't think they use it. GeyserConnect only uses the setter in GeyserSession iirc